src/com/twitter/intellij/pants/service/project/model/ProjectInfo.java

```diff
@@ -40,6 +43,9 @@ public ProjectInfo() {
   // name to info
   protected Map<String, TargetInfo> targets;
 
+  @SerializedName("available_target_types")
+  protected String[] availableTargetTypes = {};
```

Comment:
not an issue right now, but if we want to expand the features a little to include what fields a target type should include, e.g.
```
java_library( 
 dependencies = [...]
)
```

It might need to suggest that `java_libraray` should have `dependencies` as a sub field and list of string as its type.

Something to keep in mind, or maybe add as a comment.

Reply:
Added a comment about it - should be resonable to do in the future here.